modules/deploy/pages/deployment-option/cloud/managed-connectors/sizing-connectors.adoc

```diff
@@ -0,0 +1,26 @@
+= Sizing Connectors
+:description: How to choose number of tasks to set for a connector.
+:page-aliases: cloud:managed-connectors/task-count.adoc
+:page-cloud: true
+
+== What is a connector task?
+When you set up a connector, its main responsibility is to validate the provided configuration and spawn tasks.
+The actual work is performed by the connector tasks. Setting multiple tasks for a connector allows to pararelize the
+work, resulting in higher throughputs. There are a couple of things that need to be taken into consideration:
+
+- for source connectors, the possibility to increase tasks to achieve higher througput depends entirely on the specific
+connector implementation and configuration, and for many of them only 1 connector task is possible (e.g. for Debezium
+connectors only 1 task can be used).
+
+- for sink connectors, the pararelizm is achieved by evenly distributing partitions of topics configured for the
+connector between tasks. The number of partitions needs to be equal or higher than the number of tasks.
+
+== What throughput can a single task provide?
+Connector throughput depends on many factors, including converters used, compression, message size, performance of
+external system. As a rule of thumb you may expect a single connector task to provide 1-2 MB/s of throughput.
+
+== How many tasks should I set for a sink connector?
+It not easy or obvious to determine the number of tasks to use for a given workload, so you need to experiment to find
+the right number. Start with low number of tasks and wait a couple of minutes to see how it performs. Keep increasing
+the number of tasks until safisfactory throughput is achieved. Keep in mind that the underlying infrastructure needs to
+scale to provide place for more tasks. Waiting ~10 min after each change gives time for the system to scale up.
```

Comment:
```suggestion
It can be a challenge to determine the number of connector tasks to use for a given workload, so you must experiment to find the right number. Start with low number of connector tasks and wait a couple of minutes to view performance. Keep increasing the number of tasks until satisfactory throughput is achieved. Keep in mind that the underlying infrastructure must scale to provide room for additional connector tasks. Waiting roughly 10 minutes after each change should provide sufficient time for the system to scale up.
```